### PR TITLE
Dashboard: Avoid "field is never read" warnings

### LINKED ISSUE #567: Dashboard example generated with 2 `pub` declarations
The example `The Unstable Book/Language Features/trait_alias/14.rs` fails to check because there are two `pub` declarations before the `main` function. Find what the root cause is and fix it.

### PR DESCRIPTION
**Pull request by adpaco-aws**
### Description of changes: 

Declare each Litani struct field as public to remove "field is never read" warnings.

### Testing:

* How is this change tested? Running dashboard.

* Is this a refactor change? No.

### Checklist
- [x] Each commit message has a non-empty body, explaining why the change was made
- [x] Methods or procedures are documented
- [x] Regression or unit tests are included, or existing tests cover the modified code
- [ ] My PR is restricted to a single feature or bugfix

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: src/tools/dashboard/src/books.rs [hunk 1]
```diff
@@ -283,7 +283,10 @@ fn prepend_props(path: &Path, example: &mut Example, config_paths: &mut HashSet<
 
 /// Make the main function of a test public so it can be verified by rmc.
 fn pub_main(code: String) -> String {
-    code.replace("fn main", "pub fn main")
```
Comment (celinval) [inline src/tools/dashboard/src/books.rs:286]: I don't think we need this hack anymore. We should be able to verify binary crates now.

Comment (celinval) [reply src/tools/dashboard/src/books.rs:286]: https://github.com/model-checking/rmc/pull/656

Comment (adpaco-aws) [reply src/tools/dashboard/src/books.rs:286]: Thanks!! I have left a review there and hope we can merge it soon. Should we reuse this PR for the warning changes or close it and open a new one with them?

Comment (celinval) [reply src/tools/dashboard/src/books.rs:286]: We can reuse it. Maybe just make sure you edit the title!

Comment (adpaco-aws) [reply src/tools/dashboard/src/books.rs]: Updated PR to contain only the warnings changes.

**Review by celinval — APPROVED** [review]

**Pull request merged**